README.md

```diff
@@ -1,37 +1,88 @@
 
-The service has the following dependencies:
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                         | Description                                                      |
+|-------------------------------------------------|------------------------------------------------------------------|
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`            | Runs all dependencies                                            |
+| `sm2 --start CUSTOMS_EMAIL_FRONTEND_ALL `       | Runs all email dependencies                                      |
+| `sm2 -s`                                        | Shows running services                                           |
+| `sm2 --stop CUSTOMS_FINANCIALS_EMAIL_THROTTLER` | Stop the micro service                                           |
+| `sbt run` `                                     | (from root dir) to compile the current service with your changes |
+
+### Runtime Dependencies
+(These are subject to change and may not include every dependency)
 
 * `AUTH`
 * `AUTH_LOGIN_STUB`
 * `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
 * `USER_DETAILS`
 * `EMAIL`
 * `CONTACT_FRONTEND`
 
-You can use the CUSTOMS_EMAIL_THROTTLER profile in service manager to start these services.
+## Testing
+
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
+
+### Unit Tests
+
+| Command                                | Description                  |
+|----------------------------------------|------------------------------|
+| `sbt test`                             | Runs unit tests locally      |
+| `sbt "test/testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
+
+### Coverage
 
-Once these services are running, you should be able to do `sbt "run 9872"` to start in `DEV` mode.
+| Command                                  | Description                                                                                                 |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-3.3.5/scoverage-report/index.html |
 
-## Running tests
+## Available Routes
 
-There is just one test source tree in the `test` folder. Use `sbt test` to run them.
+You can find a list of microservice specific routes here - `/conf/app.routes`
 
-To get a unit test coverage report, you can run `sbt clean coverage test coverageReport`,
-then open the resulting coverage report `target/scala-2.12/scoverage-report/index.html` in a web browser.
+Application entrypoint:  `/customs/payment-records`
```

Comment:
Better to remove this as it's a backend service. You could add below if you want 

POST    /enqueue-email 

Request Body
==========

{
   "to": [
      "email1@example.co.uk",
      "email2@example.co.uk"
     ],
    "templateId": "$id",
     "parameters": {
      "param1": "value1",
       "param2": "value2"
     },
     "force": false,
    "enrolment": "testEori",
     "eventUrl": "event.url.co.uk",
     "onSendUrl": "on.send.url.co.uk"
}

Response Body
{
  "Status": "Ok",
  "message": "Email successfully queued"
}

Response code specification:
202 If the request is processed successful
400 This status code will be returned in case of incorrect data, incorrect data format, missing parameters etc are provided in the request